app/src/main/java/com/duckduckgo/app/global/view/FaviconImageView.kt

```diff
@@ -69,15 +69,27 @@ fun ImageView.loadDefaultFavicon(domain: String) {
     this.setImageDrawable(generateDefaultDrawable(this.context, domain))
 }
 
+/**
+ * Generates a default drawable which has a colored background with a character drawn centrally on top
+ * The given domain is used to generate a related background color.
+ * If an `overridePlaceholderCharacter` is specified, it will be used for the placeholder character
+ * If no `overridePlaceholderCharacter` is specified, the placeholder character will be extracted from the domain
+ */
 fun generateDefaultDrawable(
     context: Context,
     domain: String,
+    overridePlaceholderCharacter: String? = null,
 ): Drawable {
     return object : Drawable() {
         private val baseHost: String = domain.toUri().baseHost ?: ""
 
-        private val letter
-            get() = baseHost.firstOrNull()?.toString()?.uppercase(Locale.getDefault()) ?: ""
+        private val letter: String
+            get() {
```

Comment:
This is the crux of the change: being able to override the default favicon's default logic to extract the placeholder character from the host and instead allowing us to pass in exactly what should be shown (while still benefitting from all of the other stuff it's doing)